src/endGame.js

```diff
@@ -35,7 +35,9 @@ function postData(blob, filename) {
     redirect: "follow"
   };
 
-  fetch(process.env.RECORDING_SAVE_URI, requestOptions)
+  var endpoint = process.env.BASE_API_URL + "/matches";
```

Comment:
Not for this PR: In a future, given that this is a public repo, maybe the whole URL should be stored inside an ENV var

Reply:
hmm agree to disagree.

creo que lo más común es tener solo la base de la url en la env var. 

tener una env var por endpoint es too much. 

aparte, el cliente si o sí tiene que conocer de antemano cómo es el backend, si no no sabría qué cosas mandarle en el payload o qué cosas esperar en el response, así que es lo mismo hacer que el cliente que sepa también a qué path enviar el request.